### ISSUE TITLE
HU_SPCA_Incluir Agente Responsavel - Botão "Limpar"

### ISSUE THREAD

**Issue by jmatil** [islssa/SPCA#170]
Ao acionar ao botão limpar, a funcionalidade está sendo executada conforme a exigência da documentação, mas a informação de exceção ainda permanece fixa na tela. O sistema deveria limpar todos os dados incluindo as informações de alerta? Segue imagem

![5](https://cloud.githubusercontent.com/assets/6584772/7203247/9ff8594e-e4ef-11e4-8fc6-84ea9d9c6144.png)

**Comment by dassuncao** [comment]
Sim, a apresentação de mensagens de alerta não devem permanecer na tela ao limpar dados.
